source/support/resolve/index.html.md.erb

```diff
@@ -103,26 +102,26 @@ The apps communicate, so a problem in one could cascade to the other.
 ## Solving the problem
 
 GOV.UK Accounts is still part of GOV.UK, so make sure to follow the
-[main incident guidance][incident], in addition to diagnosing and
+[GOV.UK Developer documentation incident guidance][incident], in addition to diagnosing and
 fixing any accounts-specific issue.
 
 [incident]: https://docs.publishing.service.gov.uk/manual/incident-what-to-do.html
 
 ### There's a problem with the PaaS
 
-If the problem is with the PaaS itself (for example, requests aren't
+If the problem is with the PaaS itself (for example, requests are not
 even making it to the app, despite the app being healthy), you can
 escalate the issue to the PaaS on-call support.
 
-*See [the PaaS support page][paas-support] and [rota][paas-rota].*
+See [the PaaS support page][paas-support] and [rota][paas-rota] for more information.
 
 [paas-support]: https://admin.cloud.service.gov.uk/support
 [paas-rota]: https://rotas.cloudapps.digital/
 
 ### Someone is attacking us
 
-*See the section on security incidents in the [main incident
-guidance][incident].*
+See the section on security incidents in the [GOV.UK Developer Documenation incident
```

Comment:
Looks like there is a  small typo here: `Documenation` should be `Documentation`

Reply:
My mistake. Good spot.